controllers/articles.controller.js

```diff
@@ -2,9 +2,12 @@ const articlesModel = require("../models/articles.model");
 const usersModel = require("../models/users.model");
 
 exports.getArticles = (req, res, next) => {
-  articlesModel
-    .getArticlesModel()
-    .then((articles) => {
+  const { sort_by, order, topic } = req.query;
```

Comment:
because you are deconstructing `req.query` here , anything that has an incorrect key will be `undefined` 

for example if the query string is `sort_byyyyyyy` then the key `sort_by` will be undefined
because of this, the sort_by will resort to it's default value in the `getArticlesModel` , which means there isn't strictly speaking a need for the `checkArticleQueryExists`